frontend/src/components/widgets/Slider/Slider.tsx

```diff
@@ -222,8 +244,10 @@ class Slider extends React.PureComponent<Props, State> {
           aria-valuetext={formattedValue}
```

Comment:
It looks like this ref in `StyledThumb` is forwarded down from the original `sliderRef`, but I couldn’t find where it was used - any chance you remember why this is passed?

Reply:
I'm not sure if that's needed. Try removing it and see if something fails.